types/pnpapi/index.d.ts

```diff
@@ -39,3 +40,11 @@ export function resolveRequest(
 ): string | null;
 
 export function setup(): void;
+
+declare global {
+    namespace NodeJS {
+        interface ProcessVersions {
+            pnp: string;
```

Comment:
Shouldn't it still be optional? Just loading the types doesn't mean it's pnp at runtime

Reply:
Agreed 👍